HousingSearchListener/V1/Gateway/PersonApiGateway.cs

```diff
@@ -35,11 +35,12 @@ public PersonApiGateway(IHttpClientFactory httpClientFactory, IConfiguration con
         }
 
         [LogCall]
-        public async Task<Person> GetPersonByIdAsync(Guid id)
+        public async Task<Person> GetPersonByIdAsync(Guid id, Guid correlationId)
         {
             var client = _httpClientFactory.CreateClient();
             var getPersonRoute = $"{_getPersonApiRoute}/persons/{id}";
 
+            client.DefaultRequestHeaders.Add("x-correlation-id", correlationId.ToString());
```

Comment:
Could this be made a constant?

Reply:
It will hopefully be added to the header constants in the core package shortly then it will be referenced from there.